docs/design/comparison-to-olm.md

```diff
@@ -0,0 +1,35 @@
+# Operand Deployment Lifecycle Manager (ODLM) and Operator Lifecycle Manager (OLM)
+
+When it comes to deploying operator services, users will always want to know what is the difference between Operand Deployment Lifecycle Manager (ODLM) and Operator Lifecycle Manager (OLM). In this document, we compare them and discuss what advantages of using ODLM and when you should pick ODLM as an extension of OLM.
+
+## What is Operator Lifecycle Manager?
+
+Operator Lifecycle Manager (OLM) helps users install, update, and manage the lifecycle of Kubernetes native applications (Operators) and their associated services running across their OpenShift Container Platform clusters.
+It implements features, like application lifecycle management and dependency management. For more details, you can check [operator-lifecycle-manager](https://github.com/operator-framework/operator-lifecycle-manager)
+
+## How does the ODLM work?
+
+The ODLM manages an opinionated deployment of the operands for OLM managed operators.  This provides a mechanism for dynamically deploying dependent (and optionally, shared) services in a prescriptive manner and allowing these deployments to interact when and where needed in a scoped fashion.
```

Comment:
> What do you mean by opinionated?

It means `rational` here. I pick it from the ODLM feature spec. We can replace it with other clear words

> Need to consider who the target audience is. They might not know what an operand is since it's not official k8s or Openshift terminology, so need to define it.

It is a good point. I believe we don't mention what `operand` means in the README.md either.

Reply:
> The ODLM manages an opinionated (rational) deployment

Still seems unclear to me. What makes this deployment different from a regular deployment?

Reply:
Then how about just say: 
> The ODLM manages the deployment of the operands for OLM managed operators.

Reply:
yea, good with me